Gemfile

```diff
@@ -2,22 +2,22 @@ source 'https://rubygems.org'
 
 ruby IO.read('.ruby-version').strip
 
-gem 'rails', '4.2.6'
+gem 'jbuilder', '~> 2.0'
```

Comment:
Weird how rubocop wasn't smart enough to order the entries given a whitespace line. Shall we put them back together and order them all? It seems they've been split arbitrarily.